kempner_hpc_handbook/getting_started/overview_of_kempner_cluster.md

```diff
@@ -30,23 +30,21 @@ name: Diagram of HPC Cluster
 ---
 This is a diagram of the Kempner HPC cluster within the FASRC infrastructure showing the arrangement of nodes, networking, and storage systems.
 ```
-Currently, there are two GPU racks in production at the Massachusetts Green High Performance Computing Center (MGHPCC): one A100 GPU rack and one H100 GPU rack, with three more H100 GPU racks coming soon. In the following diagram, both the current and upcoming racks are shown, in addition to the network core designed for non-blocking communication between GPUs.
+Currently, there are four GPU racks in production at the Massachusetts Green High Performance Computing Center (MGHPCC): one A100 GPU rack and four H100 GPU racks. In the following diagram, the current racks are shown, in addition to the network core designed for non-blocking communication between GPUs.
 
 <p align="center">
-  <img src="https://github.com/KempnerInstitute/kempner-hpc-handbook/blob/main/kempner_hpc_handbook/getting_started/figures/svg/Clusters_1.svg" width="90%" />
+  <img src="https://github.com/KempnerInstitute/kempner-hpc-handbook/blob/main/kempner_hpc_handbook/getting_started/figures/svg/racks_1.svg" width="90%" />
 </p>
 
```

Comment:
Please use the following format for figures:

```
{figure} figures/svg/racks_1.svg
 ---
 height: 500 px
 name: Current and Upcoming GPU Racks and Network Core
 ---
 This is a diagram of the GPU racks and network core.
 ```
 
We do not use hard-coded path for the images.